test_imports.py

```
@@ -0,0 +1,35 @@
+    from audify.cli import cli
+
+    print("✓ cli imported successfully")
+except Exception as e:
+    print(f"✗ Failed to import cli: {e}")
+    traceback.print_exc()
+    sys.exit(1)
+
+try:
+    print("Importing audify.audiobook_creator...")
+    print("✓ audiobook_creator imported successfully")
+except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)